internal/b/blueprints.go

```diff
@@ -18,10 +18,16 @@ import (
 	"crypto/rand"
 	"encoding/hex"
 	"fmt"
+	"net"
 	"strconv"
 	"strings"
 )
 
+var (
+	// HostnameRunningComplement is the hostname of Complement from the perspective of a Homeserver.
+	HostnameRunningComplement = "host.docker.internal"
```

Comment:
Can't import this from `docker.HostnameRunningComplement` because then we will have a circular dependency

Reply:
That doesn't mean you should make the constant here, that means you should move the logic to where we actually care about the host/port aka `/docker`.